build-farm/signSBOM.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
This will need some changes for integrating into the build-farm CI, but we should do that in the CI PR.

Reply:
This is fine for this PR